bindings/python/test/flight/system/dynamics/__init__.py

```diff
@@ -0,0 +1,8 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Astrodynamics
+# @file           bindings/python/test/flight/system/dynamics/__init__.py
+# @author         Remy Derollez <remy@loftorbital.com>
```

Comment:
Put your name instead? (and apply to other files too)

Reply:
fixed